README.md

```diff
@@ -102,24 +100,28 @@ username and submit a pull request!
 [kmonad-contrib]: https://github.com/kmonad/kmonad-contrib
 
 ### Editor Support for the Configuration Language
+
 - [Emacs](https://github.com/kmonad/kbd-mode)
 - [Vim](https://github.com/kmonad/kmonad-vim)
```

Comment:
These (indeed, everything in 11d843180cffb045f3f8fcec1d495f3af8ee6744) seem more like stylistic choices than anything else. Probably not worth introducing so much commit/blame noise for this